src/Maestro/tests/Scenarios/arcade-update.ps1

```diff
@@ -38,11 +38,11 @@ try {
     $targetRepoUri = Get-Github-RepoUri $targetRepoName
 
     Write-Host "Creating a test channel '$testChannelName'"
-    try { Darc-Command delete-channel --name $testChannelName } catch {}
-    Darc-Add-Channel $testChannelName "test"
+    try { Darc-Command "delete-channel" "--name" "$testChannelName"  } catch {}
```

Comment:
Since Darc-Command is now taking all the values from the remaining args, the quotes around every value don't seem necessary. I believe we can remove the quotes around any string that doesn't have spaces in it. So pretty much every one except for the ones in the merge policy IgnoreChecks.

Reply:
Similarly applies to Darc-Add-Channel and Darc-Add-Subscription. The callers don't need to quote the params they pass, as the function does it for them.